sdk/eventhub/Azure.Messaging.EventHubs/stress/README.md

```diff
@@ -0,0 +1,63 @@
+# Event Hub Stress Tests
```

Comment:
I just thought of this, but can you put a link to the stress readme (https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/chaos/README.md) somewhere in here?